CollisionDetectionSystem/Interfaces/ITransponderReceiver.cs

```diff
@@ -1,13 +1,14 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace CollisionDetectionSystem
 {
-	public delegate void DataDel(TransponderData data);
+	public delegate void DataDel( List<TransponderData> data);
 
 	public interface ITransponderReceiver
 	{
-		void ReceiveData(TransponderData data);
-		void PrepareDataForPost(TransponderData data);
+		void ReceiveData(List<TransponderData> data);
+		void PrepareDataForPost( List<TransponderData> data);
 		event DataDel PostDataEvent;
```

Comment:
okay, I see, not a minute's worth but 3 elements with all the same timestamps.  That is  good. However, TransponderReceiver.ReceiveData may have to put like things together  (if that is what you need), because in real life, the transponder from our aircraft, intruder1 aircraft and intruder2 aircraft will all send out pings from their transponders individually within possibly the "same second", and then the TransponderReceiver.receiveData(transponderData) would get all those individually.   I don't think we can change that part of the design, as that is a real life constraint.

What we do from there is a design choice, but we can't pretend that 3 different aircrafts are going to assemble them for us in a list for us.(they don't know about each other necessarily)

 I realize this list makes it easier for us to test, but we can't break the product in order for it to be easier to test.  

Am I making any sense? 
In other words, I see what you are saying, but TransponderReceiver.receiveData(transponderData) has to be left like that because it will only get one aircraft reading at a time.   
If TransponderReceiver wants to assemble things as a list or just call the PostDataEvent for each transponderData element is up to you.

I know that seems like a bunch of hoop jumping, but realize that MockTransponder is an object trying to mimic what the real world would be sending us.